Apps/Examples/Examples/All Examples/SnapshotterCoreGraphicsExample.swift

```diff
@@ -11,11 +11,11 @@ public class SnapshotterCoreGraphicsExample: UIViewController, ExampleProtocol {
     override public func viewDidLoad() {
         super.viewDidLoad()
 
-        let accessToken = AccountManager.shared.accessToken!
+        let mapInitOptions = MapInitOptions()
```

Comment:
why is this example different and use a different pattern?

Reply:
Because the same resource options are being used for the snapshot too.